doc/api/figure_api.rst

```diff
@@ -5,5 +5,304 @@
 .. currentmodule:: matplotlib.figure
 
 .. automodule:: matplotlib.figure
-   :members:
-   :inherited-members:
+   :no-members:
+   :no-undoc-members:
+
+The Figure class
+================
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary_class_only.rst
+   :nosignatures:
+
+   Figure
+
+
+Adding Axes and SubFigures
+==========================
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.add_axes
+   Figure.add_subplot
+   Figure.subplots
+   Figure.subplot_mosaic
+   Figure.add_gridspec
+   Figure.get_axes
+   Figure.axes
+   Figure.delaxes
+   Figure.subfigures
+   Figure.add_subfigure
+
+Saving
+======
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.savefig
+
```

Comment:
There are 2 new lines here, before the next section. This spacing is inconsistent in other parts of the code. Some sections have 1 new line before them and some have 2. It would be good if there can be some uniformity.